smac/epm/rf_with_instances.py

```diff
@@ -28,6 +29,8 @@ class RandomForestWithInstances(AbstractEPM):
         2 dimension where the first dimension consists of 3 different
         categorical choices and the second dimension is continuous than we
         have to pass np.array([2, 0]). Note that we count starting from 0.
+    bounds: np.ndarray (D)
```

Comment:
The parameter name bounds suggests that the expected array is of shape `(D, 2)`.